# HG changeset patch
# User jwe
# Date 745014942 0
#      Tue Aug 10 20:35:42 1993 +0000
# Node ID 36bbfe8c8d9fe6a0733a4cc5c51f303710f200ce
# Parent  2cd2476fb32d813239051b9363924e44f4591a58
[project @ 1993-08-10 20:35:42 by jwe]
Use %foo%, no @foo@ to do mark things to substitute.

diff --git a/src/defaults.h.in b/src/defaults.h.in
--- a/src/defaults.h.in
+++ b/src/defaults.h.in
@@ -20,21 +20,21 @@ along with Octave; see the file COPYING.
 Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
 
 */
 
 #if !defined (_defaults_h)
 #define _defaults_h 1
 
 #ifndef DEFAULT_PAGER
-#define DEFAULT_PAGER @DEFAULT_PAGER@
+#define DEFAULT_PAGER %DEFAULT_PAGER%
 #endif
 
 #ifndef OCTAVE_HOME
-#define OCTAVE_HOME @OCTAVE_HOME@
+#define OCTAVE_HOME %OCTAVE_HOME%
 #endif
 
 #endif
 
 /*
 ;;; Local Variables: ***
 ;;; mode: C++ ***
 ;;; page-delimiter: "^/\\*" ***
